sdk/openai/openai/test/public/node/batches.spec.ts

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { matrix } from "@azure-tools/test-utils-vitest";
+import { describe, beforeEach, it } from "vitest";
+import OpenAI, { AzureOpenAI, toFile } from "openai";
+import { createClient } from "../utils/createClient.js";
+import {
+  APIVersion,
+  DeploymentInfo,
+  getDeployments,
+  maxRetriesOption,
+  withDeployments,
+} from "../utils/utils.js";
+import { FileObject } from "openai/resources/files.mjs";
+import { assertBatch } from "../utils/asserts.js";
+
+// TODO: Unskip the test when the delete file behavior is updated
+describe.skip("Batches", () => {
+  matrix([[APIVersion.Preview]] as const, async function (apiVersion: APIVersion) {
+    describe(`[${apiVersion}] Client`, () => {
+      let client: AzureOpenAI | OpenAI;
+      let file: FileObject;
+      let deployments: DeploymentInfo[] = [];
+
+      beforeEach(async function () {
+        client = createClient(apiVersion, "completions", maxRetriesOption);
+        deployments = await getDeployments("vision");
```

Comment:
shouldn't the resource be aligned between getDeployments and createClient? Also, is retrying disabled here?

Reply:
Yes, this is a mismatch that might be copied from other file. I'll update it.